mods/sp/rules/vehicles.yaml

```diff
@@ -1680,6 +1577,7 @@ DEATHCLAW:
 		FacingTolerance: 2
 
 HVRTRUK3:
+	Inherits@1: ^AutoTargetGroundAssaultMove
 	Inherits@2: ^HoverVehicle
```

Comment:
Needs to setup hold fire as the default stance.

Reply:
It has. you can find it at the end of the lines of this actor.